lib/firehose/server/message.rb

```diff
@@ -0,0 +1,9 @@
+module Firehose
+  module Server
+    class Message < Struct.new(:payload, :sequence)
```

Comment:
I tried your second version but that caused a spec failure (not sure why). I'll stick with the first version